crates/viewer/re_data_ui/src/component_path.rs

```diff
@@ -15,16 +15,26 @@ impl DataUi for ComponentPath {
     ) {
         let Self {
             entity_path,
-            component_descriptor,
+            component,
         } = self;
 
         let engine = db.storage_engine();
 
+        let Some(component_descriptor) = engine
+            .store()
+            .entity_component_descriptor(entity_path, *component)
+        else {
+            ui.label(format!(
+                "Entity {entity_path:?} has no component {component:?}"
+            ));
+            return;
+        };
```

Comment:
I guess these `else` clauses are there to mostly handle broken links that were shared?

We could use `ui.warning_label(...)` to draw more attention.